tests/unit_tests/test_data_kalbach_mann.py

```diff
@@ -75,114 +67,74 @@ def test_atomic_representation(neutron, triton, b10, c12, c13, na23):
+        _AtomicRepresentation(z=5, a=-2)
+    with pytest.raises(ValueError):
         neutron - triton
 
 
-def test__calculate_separation_energy(triton, b10, c13):
+def test_separation_energy(triton, b10, c13):
     """Comparison to hand-calculations on a simple example."""
-    assert _calculate_separation_energy(
+    assert _separation_energy(
         compound=c13,
         nucleus=b10,
         particle=triton
     ) == pytest.approx(18.6880713)
 
 
-def test__return_entrance_channel_energy():
-    """Comparison to hand-calculations on a simple example."""
-    assert _return_entrance_channel_energy(
-        e_p=5.2,
-        awr_t=13.7,
-        awr_p=7.2
-    ) == pytest.approx(3.4086124)
-
-
-def test__return_emission_channel_energy():
-    """Comparison to hand-calculations on a simple example."""
-    assert _return_emission_channel_energy(
-        e_e=6.8,
-        awr_r=49.2,
-        awr_e=5.4
-    ) == pytest.approx(7.5463415)
-
-
-def test__calculate_kalbach_slope(neutron, triton, b10, c12, c13):
-    """Comparison to hand-calculations for n + c12 -> c13 -> triton + b10."""
-    energy_projectile = 10.2  # [eV]
-    energy_emitted = 5.4  # [eV]
-
-    assert _calculate_kalbach_slope(
-        energy_projectile=energy_projectile,
-        energy_emitted=energy_emitted,
-        projectile=neutron,
-        target=c12,
-        compound=c13,
-        emitted=triton,
-        residual=b10
-    ) == pytest.approx(0.8409921475)
-
-
-def test_return_kalbach_slope():
+def test_kalbach_slope():
     """Comparison to hand-calculations for n + c12 -> c13 -> triton + b10."""
     energy_projectile = 10.2  # [eV]
     energy_emitted = 5.4  # [eV]
 
     # Check that NotImplementedError is raised if the projectile is not
     # a neutron
     with pytest.raises(NotImplementedError):
-        return_kalbach_slope(
+        kalbach_slope(
             energy_projectile=energy_projectile,
             energy_emitted=energy_emitted,
-            iza_projectile=1000,
-            iza_emitted=1,
-            iza_target=6012
+            za_projectile=1000,
+            za_emitted=1,
+            za_target=6012
         )
 
-    assert return_kalbach_slope(
+    assert kalbach_slope(
         energy_projectile=energy_projectile,
         energy_emitted=energy_emitted,
-        iza_projectile=1,
-        iza_emitted=1003,
-        iza_target=6012
+        za_projectile=1,
+        za_emitted=1003,
+        za_target=6012
     ) == pytest.approx(0.8409921475)
 
 
 @pytest.mark.parametrize(
-    "hdf5_filename, endf_type, endf_filename", [
-        ('O16.h5', 'neutrons', 'n-008_O_016.endf'),
-        ('Ca46.h5', 'neutrons', 'n-020_Ca_046.endf'),
-        ('Hg204.h5', 'neutrons', 'n-080_Hg_204.endf')
+    "hdf5_filename, endf_filename", [
+        ('O16.h5', 'n-008_O_016.endf'),
+        ('Ca46.h5', 'n-020_Ca_046.endf'),
+        ('Hg204.h5', 'n-080_Hg_204.endf')
     ]
 )
 def test_comparison_slope_hdf5(hdf5_filename, endf_type, endf_filename):
```

Comment:
```suggestion
def test_comparison_slope_hdf5(hdf5_filename, endf_filename):
```
The tests won't pass without this correction.